Makefile

```diff
@@ -57,17 +58,7 @@ docker-all: docker
 
 .PHONY: docker-push
 docker-push:
-	docker push $(REGISTRY)/oauth2-proxy:latest
-
-.PHONY: docker-push-all
-docker-push-all: docker-push
-	docker push $(REGISTRY)/oauth2-proxy:latest-amd64
-	docker push $(REGISTRY)/oauth2-proxy:${VERSION}
-	docker push $(REGISTRY)/oauth2-proxy:${VERSION}-amd64
-	docker push $(REGISTRY)/oauth2-proxy:latest-arm64
-	docker push $(REGISTRY)/oauth2-proxy:${VERSION}-arm64
-	docker push $(REGISTRY)/oauth2-proxy:latest-armv6
-	docker push $(REGISTRY)/oauth2-proxy:${VERSION}-armv6
+	docker buildx build --push --platform linux/amd64,linux/arm64,linux/arm/v6 -t $(REGISTRY)/oauth2-proxy:latest .
```

Comment:
Where did we get to on keeping the old tag structure during the push for a period?

Reply:
ah, right! Sorry about that

Reply:
As far as I can tell, this still isn't resolved? Does that match your understanding?

Reply:
It might be confusing if you're looking here only, if you look at the file review, the `docker-push-all` target looks like main. 
Am I missing something?

main:

```
.PHONY: docker-push-all
docker-push-all: docker-push
	docker push $(REGISTRY)/oauth2-proxy:latest-amd64
	docker push $(REGISTRY)/oauth2-proxy:${VERSION}
	docker push $(REGISTRY)/oauth2-proxy:${VERSION}-amd64
	docker push $(REGISTRY)/oauth2-proxy:latest-arm64
	docker push $(REGISTRY)/oauth2-proxy:${VERSION}-arm64
	docker push $(REGISTRY)/oauth2-proxy:latest-armv6
	docker push $(REGISTRY)/oauth2-proxy:${VERSION}-armv6
```

This branch:

```
.PHONY: docker-push-all
docker-push-all: docker-push
	docker push $(REGISTRY)/oauth2-proxy:latest-amd64
	docker push $(REGISTRY)/oauth2-proxy:${VERSION}
	docker push $(REGISTRY)/oauth2-proxy:${VERSION}-amd64
	docker push $(REGISTRY)/oauth2-proxy:latest-arm64
	docker push $(REGISTRY)/oauth2-proxy:${VERSION}-arm64
	docker push $(REGISTRY)/oauth2-proxy:latest-armv6
	docker push $(REGISTRY)/oauth2-proxy:${VERSION}-armv6
```
https://github.com/oauth2-proxy/oauth2-proxy/blob/8b085cd48050d8e6855cb62f4b10d11748acadf2/Makefile#L63-L71